File: from-0001-to-0100/0075-sort-colors/solution.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        if not nums:
            return nums
        
        c = Counter(nums)
        i = 0
        for k in (0, 1, 2):
            if k in c:
                for _ in range(c[k]):
                    nums[i] = k
                    i += 1
                    

File: from-0001-to-0100/0075-sort-colors/solution.go
func sortColors(nums []int)  {
    count := make(map[int]int)
    for _, num := range nums {
        count[num] += 1
    }
    i := 0
    for j := 0; j < 3; j ++ {
        for k := 0; k < count[j]; k ++ {
            nums[i] = j
            i += 1
        }
    }
}